lib/heroku_pg_backups_archive/toolbelt_helper.rb

```diff
@@ -17,6 +17,11 @@ def fetch_backup_info(backup_id)
 
       private
 
+      def follower_db(app)
+        output = HerokuHelper.run_heroku_command("pg:info --app #{app} | grep Followers | head -n 1")
+        return output.blank? ? "" : output.split(" ").last
+      end
+
```

Comment:
@cspeper couple questions

- thoughts on providing the database name as a configuration variable (... versus trying to parse command output)
- does this work when there are multiple followers?
- do we have a test for this?